src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int SIZE_OF_THE_ARRAY = 10;
+
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[SIZE_OF_THE_ARRAY];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < SIZE_OF_THE_ARRAY / 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
+            }
+            figures[i].toDraw();
```

Comment:
The method `toDraw()` should be renamed to better reflect its purpose. According to the checklist, method names should not be verbs. Consider renaming it to something like `draw()` to align with the checklist's recommendation on naming conventions.